params/types/coregeth/chain_config.go

```diff
@@ -210,6 +210,8 @@ type CoreGethChainConfig struct {
 	BlockRewardSchedule         ctypes.Uint64BigMapEncodesHex `json:"blockReward,omitempty"`          // JSON tag matches Parity's
 
 	RequireBlockHashes map[uint64]common.Hash `json:"requireBlockHashes"`
+
+	Lyra2NonceTransitionBlock *big.Int `json:"-"`
```

Comment:
```suggestion
	Lyra2NonceTransitionBlock *big.Int `json:"lyra2NonceTransitionBlock,omitempty"`
```

I think you want this to be included in JSON. The configuration is stored with genesis in the database as JSON and fills the runtime configuration on startup if no flag is set. An unwritten value will cause its omission in this case.

This also applies to the other configuration structs.

Reply:
You're right, thank you for noticing it.